jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -658,71 +658,73 @@ ecma_op_general_object_define_own_property (ecma_object_t *obj_p, /**< the objec
   ecma_property_types_t current_type = ECMA_PROPERTY_GET_TYPE (current_p);
   const bool is_current_data_descriptor = (current_type == ECMA_PROPERTY_TYPE_NAMEDDATA);
   const bool is_current_accessor_descriptor = (current_type == ECMA_PROPERTY_TYPE_NAMEDACCESSOR);
+  const bool current_property_configurable = ecma_is_property_configurable (current_p);
```

Comment:
Please follow the naming convention and give a name like the other two bool variables above. Ex.: `is_current_property_configurable` or `is_curr_prop_configurable`